grr/server/grr_response_server/gui/ui/components/approval/approval_test.ts

```diff
@@ -19,6 +19,11 @@ function makeClient(args: Partial<Client> = {}): Client {
     clientId: 'C.1234',
     fleetspeakEnabled: true,
     knowledgeBase: {},
+    users: [],
```

Comment:
(optional) It's not part of your logic, but you'll get bonus points for reusing `newClient()` from `model_test_util.ts` here.